### ISSUE TITLE
Game wont load after applying DLL

### ISSUE THREAD

**Issue by LFFPicard** [xeropresence/AOPP#4]
Hi,

So I downloaded the DLL from the release. Put it in the folder and then tried to load Anarchy.exe.
The launcher loads and then I click play, the launcher closes and nothing happens.

See below logs, I can not see any errors at all. It just wont do anything. If I remove the DLL the game works fine again.

[Anarchy.exe.log](https://github.com/xeropresence/AOPP/files/15150530/Anarchy.exe.log)
[AnarchyOnline.exe.log](https://github.com/xeropresence/AOPP/files/15150531/AnarchyOnline.exe.log)

**Comment by xeropresence** [comment]
Are you using the old client or the new one?

 You can try opening the config.json and toggling the largeaddressaware patch. There was some funkyness with getting it to work the last time i resubscribed.

**Comment by LFFPicard** [comment]
That worked thanks!

Annoyingly I think that was the thing I wanted the most as I believe that's the patch that stops ICC crashing haha.
I will see how I get on now.

Thanks!

**Comment by xeropresence** [comment]
Are you using dgVooDoo?

**Comment by LFFPicard** [comment]
No, just base install and then this dll. Not got as far as experimenting with dgVooDoo yet

**Comment by Lopnk** [comment]
I had the same issue. Disabling the same thing let the game launch. Old client @ 1440p

**Comment by xeropresence** [comment]
I'll have to think about a workaround. I'm not sure why it worked well in the past but has issues now.

**Comment by xeropresence** [comment]
So i'm taking another look at this, on a fresh install of win10 and win11 the default setup works just fine. Are you using any sort of antivirus or anything like that?

**Comment by LFFPicard** [comment]
All I have is windows built in, no other "bells and whistles" software running. The PC is about 6months old so relatively fresh install of Windows 11.
But saying that, I tried the large address aware program as well and it crashes AO as well. So no idea what the cause is. It's been stable for a couple weeks now so I will poke around some more if/when it happens again

**Comment by Lopnk** [comment]
Any update on this  by chance? seems to still be an issue at least for me.
Win11 
turning off largeaddressaware patch lets me launch the game but will not launch without the use of AO#quicklauncher with it enabled as it bypasses the launcher

**Comment by xeropresence** [comment]
So I had something similar happen to me in another project, it lead to me reproducing the issue and was able to figure out it was some weirdness with the CWD not being set properly.

I've pushed a new build https://github.com/xeropresence/AOPP/releases/tag/v0.8.0 please let me know if it resolves the issue for you.
